### PR TITLE
Added Note on #1593

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding the important note for issue #1593

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md [hunk 1]
```diff
@@ -175,7 +175,9 @@ This rule blocks the following file types from launching unless they either meet
 >[!NOTE]
 >You must [enable cloud-delivered protection](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-antivirus/enable-cloud-protection-windows-defender-antivirus) to use this rule.
 
-Intune name: Executables that don't meet a prevalence, age, or trusted list criteria
+>[!IMPORTANT] The rule **Block executable files from running unless they meet a prevalence, age, or trusted list criterion** with GUID 01443614-cd74-433a-b99e-2ecdc07bfc25, it's owned by microsoft and is not specified by admins. It uses Microsoft CLoud's Protection to update its trusted list regularly.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: >[!IMPORTANT] The rule **Block executable files from running unless they meet a prevalence, age, or trusted list criterion** with GUID 01443614-cd74-433a-b99e-2ecdc07bfc25 **is** owned by **M**icrosoft and is...

Please do changes in bold (remove comma, change it's to is, and capitalize the M in Microsoft).

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md [hunk 2]
```diff
@@ -175,7 +175,9 @@ This rule blocks the following file types from launching unless they either meet
...
+>You can specify individual files or folders (using folder paths or fully qualified resource names) but you can't specify which rules or exclusions apply to.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: >You can specify individual files or folders (using folder paths or fully qualified resource names) but you can't specify which rules or exclusions **to apply**.

Please do change in bold (to apply)

Comment (nenonix) [reply windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: agree

Comment (jdeckerms) [reply windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: If you look at the related link, you'll see that the proper way to refer to it is "cloud-delivered protection". It's not "Microsoft CLoud's Protection", particularly not with a double-capital for cloud nor a possessive.

@andreabichsel Could you confirm the change with your product group before it's merged?

Comment (j0rt3g4) [reply windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: @jdeckerms (that was a typo on the L).

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md (hunk 1 above)
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: It uses **cloud-delivered protection** to update its trusted list regularly.

Comment (j0rt3g4) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: IT must contain the word "admin" or "admins" to people to understand in my opinion.

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md (hunk 2 above)
Comment (j0rt3g4) [reply windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: > > You can specify individual files or folders (using folder paths or fully qualified resource names) but you can't specify which rules or exclusions **to apply**.
> 
> Please do change in bold (to apply)

IT must contain the word "admin" or "admins" to people to understand in my opinion.

**Comment by j0rt3g4** [comment]
@Justinha Also it was adding that the specifics of the rule  **Block executable files from running unless they meet a prevalence, age, or trusted list criterion** uses  **cloud-delivered protection** and I named it another way, that's the fix :)

**Comment by Justinha** [comment]
@j0rt3g4 Thanks for explaining. We can probably make this point better in this topic: https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-exploit-guard/customize-attack-surface-reduction

I can move it on my side. And we can add why cloud-delivered protections is required in the notes here: https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard#block-executable-files-from-running-unless-they-meet-a-prevalence-age-or-trusted-list-criterion 

let me confirm with engineering and I'll follow up.

**Comment by j0rt3g4** [comment]
@Justinha  I see, do you want me to add that information ? :)

**Pull request merged**